### PR TITLE
Exports function that returns CouchDB version

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

ServerChecks shared lib now exports a function that returns the CouchDB version.

medic/medic-webapp#5076

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
To be used with https://github.com/medic/medic-webapp/pull/5111

**Comment by dianabarsan** [comment]
Hey, @garethbowen . Please take a look. Thanks!

File: shared-libs/server-checks/src/checks.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const couchDbVersionCheck = (serverUrl) => {
         return reject(err);
       }
       console.log(`CouchDB Version: ${body.version}`);
-      resolve();
+      resolve(body.version);
```
Comment (garethbowen) [inline shared-libs/server-checks/src/checks.js]: This means every time the code calls `getCouchDbVersion` we'll get the version output in the log. While you're currently only calling it once per start I can see this function being used more. Can we split it so there's an exported function that just returns the version, and a "private" function that does the logging?

While we're here, shouldn't `couchDbVersionCheck` throw an error if the version is < 2.0? Maybe that's a separate issue...

Comment (dianabarsan) [reply shared-libs/server-checks/src/checks.js]: I pulled the function out separately. 
I'm not sure what should be done about the actual version "check". The shared-lib contained an exact copy of the code that existed in API's `server.js`. I think it should be a separate issue, at least to describe which version we're supporting at minimum. Maybe we will want to minimally support `2.3.1` (after release)?

**Pull request merged**